plopfile.js

```diff
@@ -1,96 +0,0 @@
-const prompts = {
```

Comment:
Wow I didn't know we had this! Why remove it - does literally nobody use it?

Reply:
I think it's outdated now and I don't think anyone uses it.